src/utils.cpp

```diff
@@ -45,6 +45,10 @@ string toNodeHandle(handle nodeHandle)
     return string(base64Handle);
 }
 
+string toNodeHandle(NodeHandle nodeHandle)
+{
+    return toNodeHandle(nodeHandle.as8byte());
```

Comment:
If you plan to replace the old code, perhaps this method should not depend on the old code, but the other way round.

Reply:
I'm sure we'll do that eventually, but for this initial PR surely it's better to (a) leave the working code alone (even though it is very simple) and (b) keep the number of lines changed to a minimum